tests/cmds/test_sim.py

```diff
@@ -117,10 +118,16 @@ def _test_farm_and_revert_block(runner: CliRunner, address: str, simulator_name:
     assert "Height:          6" in five_blocks_check.output
 
     # do a reorg, 3 blocks back, 2 blocks forward, height now 8
-    reorg_result: Result = runner.invoke(cli, ["dev", "sim", "-n", simulator_name, "revert", "-b", "3", "-n", "2"])
-    assert reorg_result.exit_code == 0
-    assert "Block: 3 and above " in reorg_result.output and "Block Height is now: 8" in reorg_result.output
-
+    try:  # we do a 'try' here because creating a couple of blocks might lead to a timeout on a slow runner.
+        reorg_result: Result = runner.invoke(cli, ["dev", "sim", "-n", simulator_name, "revert", "-b", "3", "-n", "2"])
+        assert reorg_result.exit_code == 0
+        assert "Block: 3 and above " in reorg_result.output and "Block Height is now: 8" in reorg_result.output
+    except AssertionError:
+        # if we get a timeout, we wait 5 seconds then check the block height
+        time.sleep(5)
+        reorg_fallback: Result = runner.invoke(cli, ["dev", "sim", "-n", simulator_name, "status"])
+        assert reorg_fallback.exit_code == 0
+        assert "Height:          8" in reorg_fallback.output
```

Comment:
there's no coverage for this in the tests, which would seem fine given that this is an `AssertionError`. But we actually *swallow* this error and don't necessarily fail the test, which looks super shady. How can you know the assertion isn't a serious failure that is caused by a bug? In fact, aren't all assertion failures caused by bugs?